.github/workflows/nomad.yml

```
@@ -0,0 +1,201 @@
+          ExecReload=/bin/kill -HUP $MAINPID
+          ExecStart=/usr/bin/nomad agent -dev
+          KillMode=process
+          KillSignal=SIGINT
+          LimitNOFILE=65536
+          LimitNPROC=infinity
+          Restart=on-failure
+          RestartSec=2
+
+          TasksMax=infinity
+          OOMScoreAdjust=-1000
+
+          [Install]
+          WantedBy=multi-user.target
+          " >> nomad.service
+          sudo mv nomad.service /etc/systemd/system/nomad.service
+          sudo systemctl enable nomad
+          sudo systemctl start nomad
+          sudo systemctl status nomad
+          a=0
+          while [ "$a" -lt 10 ];
+          do
+            if [ $(sudo systemctl is-active nomad.service) == "active" ];
+            then 
+              echo "Nomad service is running.";
+              break;
+            else 
+              sleep 10;
+            fi
+            a=`expr $a + 1`
+          done
+          if [ "$a" -eq 10 ];
+          then
+            echo "Fail to start Nomad service";
+            sudo systemctl disable nomad;
+            sudo systemctl stop nomad;
+            sudo rm /etc/systemd/system/nomad.service;
+            exit 1;
+          fi
+
+      - name: Start Consul Service.
+        run: |
+          sudo touch /etc/systemd/system/consul.service
+          sudo echo "
+          [Unit]
+          Description=Consul
+          Documentation=https://www.consul.io/docs
+          Wants=network-online.target
+          After=network-online.target
+
+          [Service]
+          ExecReload=/bin/kill -HUP $MAINPID
+          ExecStart=/usr/bin/consul agent -dev
+          KillMode=process
+          KillSignal=SIGINT
+          LimitNOFILE=65536
+          LimitNPROC=infinity
+          Restart=on-failure
+          RestartSec=2
+
+          TasksMax=infinity
+          OOMScoreAdjust=-1000
+
+          [Install]
+          WantedBy=multi-user.target
+          " >> consul.service
+          sudo mv consul.service /etc/systemd/system/consul.service
+          sudo systemctl enable consul
+          sudo systemctl start consul
+          a=0
+          while [ "$a" -lt 10 ];
+          do
+            if [ $(sudo systemctl is-active consul.service) == "active" ];
+            then 
+              echo "Consul service is running.";
+              break;
+            else 
+              sleep 10;
+            fi
+            a=`expr $a + 1`
+          done
+          if [ "$a" -eq 10 ];
+          then
+            echo "Fail to start Consul service";
+            sudo systemctl disable consul;
+            sudo systemctl stop consul;
+            sudo rm /etc/systemd/system/consul.service;
+            exit 1;
+          fi
+
+      - name: Deploy nomad Jobs.
+        run: |
+          sudo apt-get install net-tools;
+          until nc -w 10 127.0.0.1 4646; do sleep 1; done
+          sed -i 's/<SPLUNK_ACCESS_TOKEN>/12345/g' otel-agent.nomad
+          sed -i 's/<SPLUNK_REALM>/us0/g' otel-agent.nomad
```

```suggestion
          sed -i 's/<SPLUNK_REALM>/test/g' otel-agent.nomad
```
Let's set the realm to something fake so we don't spam our production realms.

---

Changes made.